### DISCUSSION TITLE
Angular 13 Compat

### DISCUSSION THREAD

**Discussion by siddharth1903** [vmware/clarity#6485] [Q&A] [answered]
Hey guys,
Have you thought of any release date to make clr/ng compatible with angular 13? Also i believe cds/ng is automatically compatible with 13, since it just contains typings on top of web components? Asking this because we have both cds and clr angular in our app and are migrating component by component as and when things move.

Thanks in advance,
Sid

**Comment by bbogdanov** [comment, marked as answer]
Hi @siddharth1903,

We are working on it. We had some difficulties with the the latest TypeScript version and we postpone the release because of them. We are in final stages to merge #6455

Because lot of us are on PTO and US are off this Thursday and Friday because of Thanksgiving I believe we are going to do the release next week.

**Reply by siddharth1903** [reply]
Thanks much @bbogdanov .. just wanted to understand this cos we have to submit an internal plan to move too... This answers... Wish the team a happy vacation 🙂
My question on cds angular is relevant? Is it already compatible with 13 or cds angular gets an update too?

**Reply by bbogdanov** [reply]
@siddharth1903 Last time I checked everything was working fine. I'll prefer to double check, tho. I'll write back to you tomorrow

**Reply by siddharth1903** [reply]
Sure thanks

**Reply by bbogdanov** [reply]
@siddharth1903 It should be fine. Let us know if you face any issues

**Reply by ReissZachary** [reply]
@bbogdanov I am not able to add clarity to my ng 13 project still. Looks like I am getting an upstream dependency conflict. Wanted to check and see if this is wrong on my end or on Clarity end. Screenshot added. 
![image](https://user-images.githubusercontent.com/45380236/144995554-d9495262-c365-43e0-aa4a-b17f79055a1f.png)

**Reply by bbogdanov** [reply]
@ReissZachary `@clr/angular` 13 is not released, yet. We are going to release it tomorrow if everything goes smoothly

**Reply by ReissZachary** [reply]
@bbogdanov sounds good I will circle back tomorrow. Thanks!

**Reply by ReissZachary** [reply]
@bbogdanov how did release of @clr/angular 13 go? I still seem to be seeing that error on my end so I thought I would check in with you

**Reply by gnomeontherun** [reply]
The Clarity Angular 13 beta release is live, so try `ng update @clr/angular@next` to get the latest installed into your CLI project. There were issues with TypeScript breaking changes across the Angular ecosystem, so it will help us to know if there are any other issues here.

**Reply by AllForNothing** [reply]
@gnomeontherun When will you publish @clr/angular13 to the npm registry?

**Reply by mathisscott** [reply]
Clarity Angular for Angular v13 is up on npm. The specific version of `@clr/angular` is `13.0.0-next.0`. It's listed with the `next` tag for `@clr/angular`.

https://www.npmjs.com/package/@clr/angular
